buildlib/pr/main.yml

```diff
@@ -1,8 +1,11 @@
 resources:
   containers:
     - container: centos7
-      image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos7:3
+      image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos7:4
       options: -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools
+    - container: centos7_cuda11
+      image: rdmz-harbor.rdmz.labs.mlnx/ucx/centos7-mofed5.4-cuda11:1
+      options: --net=host --gpus all --ulimit memlock=-1:-1 -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools --device=/dev/infiniband/
```

Comment:
1. --net=host works?
2. why need `--ulimit memlock=-1:-1 -v /hpc/local:/hpc/local -v /auto/sw_tools:/auto/sw_tools --device=/dev/infiniband/` only for this container and not all other as well?

Reply:
1 - No, we have hack agents
2 -  it need for same test, we cannot add different options for different stages

Reply:
1- so can remove it?
2 - question is why not add these parameters to all cuda dockers below (btw is it possible to make it templated some how to not repeat the same options for every container) ?

Reply:
1 - No , it use in hack script, if script find --net=host, script remove --net=<azure_net>
2 - will try via variable, maybe azure will not fails